x-pack/plugins/apm/public/services/rest/observability_dashboard.ts

```diff
@@ -47,7 +47,8 @@ export const fetchLandingPageData = async (
           'xpack.apm.observabilityDashboard.stats.transactions',
           { defaultMessage: 'Transactions' }
         ),
-        value: sum(transactionCoordinates.map((coordinates) => coordinates.y)),
+        value:
+          mean(transactionCoordinates.map((coordinates) => coordinates.y)) || 0,
```

Comment:
Thanks for the heads up @smith , I added a condition to check if the array is not empty and to return 0 in case `corrdinate.y` is undefined. https://github.com/elastic/kibana/pull/70336/commits/ffedc7f65c3498ac75dcde83c2d04ef1fd635765#diff-cf9933de7395c67ff230849c1c748111R50-R54

Reply:
That will return 0 if any y value is non-numeric, which is fine if that's the behavior you want. When I ran into this I did `values.map(value => value.y).filter(y => isFinite(y))`. 

 (I can't remember if the data I was looking at had `null` or `NaN` or `undefined`, but let's say null for now) The first method makes it 0 if any null are found and the second gives you the mean of the the non-null values. I suppose you could also do `values.map(value => value.y ?? 0)`, which would convert all the null values to zeroes and give you the mean of that.